spec.bs

```diff
@@ -2365,10 +2365,11 @@ or "component-auction", a [=currency tag=] |componentAuctionExpectedCurrency|, a
   contributions, and some special handling of private aggregation requests involving reject-reason);
   as it would be problematic to report a winner that didn't actually win.
 
-  1. If |reportingContext|'s [=reporting context/debug reporting info=][|generatedBid|'s [=generated
-    bid/reporting id=]] does not [=map/exist=], set it to a new [=bid debug reporting info=].
+  1. Let |reportingId| be |generatedBid|'s [=generated bid/reporting id=].

```

Comment:
may not this PR, but probably better to rename `reporting id` to `reporting key` in a future PR, to avoid confusion with reporting ids like `buyer reporting id`.